routes/celebrities.routes.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const router = express.Router();
+const Celebrity = require("./../models/Celebrity.model");
+
+// llamada para obtener la vista del formulario 
+router.get('/create', (req, res, next) => {
+    res.render('celebrities/new-celebrity');
+})
+//llamada para crear actores para poner en la BBDD
+router.post('/create', (req, res, next) => {
+    const { name, occupation, catchPhrase } = req.body;
+    Celebrity
+    .create(req.body)
```

Comment:
```suggestion
router.post('/create', (req, res, next) => {
    const { name, occupation, catchPhrase } = req.body;
    Celebrity
    .create({name, occupation, catchPhrase)
```

Deconstruimos el req.body precisamente para no pasarselo a pelo al create, eso supone un riesgo de seguridad.